python/tvm/relay/frontend/pytorch.py

```diff
@@ -2120,14 +2146,17 @@ def _get_constant(node):
         attr_name = attribute_names[0]
         ty = node.output().type().kind()
 
-        if ty in ["IntType", "BoolType"]:
+        if ty == "IntType":
             return node.i(attr_name)
+        elif ty == "BoolType":
+            return bool(node.i(attr_name))
         elif ty in ["FloatType", "LongType"]:
             return node.f(attr_name)
         elif ty in ["TensorType", "CompleteTensorType"]:
             tensor = node.t(attr_name)
             if len(tensor.shape) == 0:  # tensor(0.1)
-                return float(tensor)
+                # TODO(t-vi): When is this needed?
```

Comment:
I don't really remember where I got this, maybe this one but not sure.
https://github.com/apache/incubator-tvm/blob/a14a98e845f7d076e3a060442c65cfd14216f54c/tests/python/frontend/pytorch/test_forward.py#L276

Reply:
I was planning to look at the TODOs in more detail, but it seems that the patch is rather large already...